management-servlet-api/src/test/scala/com/gu/management/servlet/BodyCachingRequestWrapperTest.scala

```diff
@@ -94,11 +97,12 @@ class BodyCachingRequestWrapperTest extends Specification {
 
   def createWrapperForBodyString(requestBody: String) = {
     BodyCachingRequestWrapper(new MockHttpServletRequest("/foo") {
-      override def getInputStream = new MockServletInputStream(new ByteArrayInputStream(requestBody.getBytes("UTF-8")))
+      override def getInputStream = new MockServletInputStream(new ByteArrayInputStream(requestBody.getBytes(utf8)))
 
       charEncoding = "UTF-8"
       contentType = "application/x-www-form-urlencoded"
     });
   }
 
+  val utf8 = Charset.forName("UTF-8")
```

Comment:
In today's fun times, trying to acquire the `Charset` in parallel tests lead to a deadlock.